packages/field-group-react/src/FieldGroup.tsx

```diff
@@ -7,12 +7,17 @@ interface Props {
     className?: string;
     helpLabel?: string;
     errorLabel?: string;
+    variant?: "secondary" | "small";
```

Comment:
Yeah, we should have this a single place

Reply:
Any ideas on where/how? Maybe a `data-types.ts` file in `jkl-core/src/utils`?